sql/updates/world/3.3.5/2021_04_24_00_world.sql

```diff
@@ -0,0 +1,12 @@
+-- delete non generic spell script for wild wyrm
+DELETE FROM `spell_script_names` WHERE `ScriptName`= 'spell_eject_passenger_wild_wyrm';
```

Comment:
Looks harmless enough to begin with, but this spell is actually used in Storm Peaks scripts:

https://github.com/TrinityCore/TrinityCore/blob/d5f1fefe7f6772b11e9fbb052bfd36be9bda98e2/src/server/scripts/Northrend/zone_storm_peaks.cpp#L1070-L1100

https://github.com/TrinityCore/TrinityCore/blob/d5f1fefe7f6772b11e9fbb052bfd36be9bda98e2/sql/old/3.3.5a/world/62_2017_04_18/2017_02_25_00_world.sql#L20-L33

If you remove that spell ScriptName without restoring a new spell ScriptName, you will find that [Quest 13003, **Thrusting Hodir's Spear**](https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=13003/thrusting-hodirs-spear), becomes a problem.
- https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=13003/thrusting-hodirs-spear

Reply:
Check changes